src/config/dynamic-theme-fixes.config

```diff
@@ -12882,6 +12882,24 @@ body {
 
 ================================
 
+wiki.teamfortress.com
+
+INVERT
+.mw-wiki-logo
+
+CSS
+html, body {
+    background-color: var(--darkreader-neutral-background) !important;
+}
```

Comment:
I don't see any need for this.

Reply:
This is to prevent this from happening.
![image](https://user-images.githubusercontent.com/31318219/123549257-62570800-d792-11eb-928a-8bdc8846ac72.png)

Reply:
That's expected to happen on site that don't set a background-color on their `HTML` element.

But now we will be overriding the body element as well which has a correct background-color.

Reply:
What do you suggest? Remove this rule and leave this as is?

Reply:
Let the rule as-is.
```suggestion
html,
body {
    background-color: var(--darkreader-neutral-background) !important;
}
```